pkg/atmos/aws-component-helper/atmos.go

```
@@ -1,108 +1,291 @@
-}
+	atmosOptions := atmos.WithDefaultRetryableErrors(ts.t, result)
 
-func destroyComponent(t *testing.T, suite *TestSuite, componentName string, stackName string, vars map[string]interface{}) (*atmos.Options, string, error) {
-	options := GetAtmosOptions(t, suite, componentName, stackName, vars)
-	out, err := atmos.DestroyE(t, options)
+	return atmosOptions
+}
 
-	return options, out, err
+// Helper to parse a map recursively
+func parseMap(m map[string]interface{}) (map[string]interface{}, error) {
```

golangci-lint (1.62.2)

func `parseMap` is unused

(unused)